index.js

```diff
@@ -5229,6 +5229,8 @@ export async function submitBom(args, bomContents) {
     autoCreate: "true",
     bom: encodedBomContents
   };
+  if (typeof(args.parentProjectId) !== "undefined") {
+    bomPayload.parentUUID = args.parentProjectId
```

Comment:
Closing bracket might be missing. Could you give it a quick test as well?

Reply:
Also add semi-colon at the end.

Reply:
Fixed